h/api/storage.py

```diff
@@ -189,8 +189,12 @@ def delete_annotation(request, id):
     :param id: the annotation id
     :type id: str
     """
-    annotation = models.elastic.Annotation.fetch(id)
-    annotation.delete()
+    if _postgres_enabled(request):
+        annotation = fetch_annotation(request, id, _postgres=True)
```

Comment:
Passing `_postgres=True` isn't strictly necessary here as the postgres feature flat will always be on when we hit this line so `fetch_annotation()` would fetch from Pg anyway, but no harm in being explicit.